settings/src/script.js

```diff
@@ -81,24 +96,12 @@ function Main() {
 			}
 
 			<div className={ 'wporg-2fa__' + screen }>
-				<CurrentScreen clickScreenLink={ clickScreenLink } />
+				<CurrentScreen clickScreenLink={ clickScreenLink } userId={ userId } userData={ userData } />
```

Comment:
```suggestion
				<CurrentScreen clickScreenLink={ clickScreenLink } userData={ userData } />
```

This doesn't seem necessary, since we're also passing `userData.id`. Not a big deal, though.

`EmailAddress` would need to be updated to use `userData.id` if this is accepted.

Reply:
Alternatively; we shouldn't pass userData at all, as elements cannot change/update when the record is updated, unless they're being re-rendered. So it's safer to always retrieve the user object using `useEntityRecord()`.

`userData` was kept there for the `<AccountStatus />` which had no reason to retrieve an entire user object if it was known already since it's only displaying data and not allowing edits. I should just update that to accept a userId though I think.